steps/lambdatest-upload/0.0.1/step.yml

```diff
@@ -0,0 +1,80 @@
+title: lambdatest-upload
```

Comment:
This is the name of the step which is present on Workflow editor UI. Should be a descriptive name for instance `LambdaTest Upload`